scripts/mytop.sh

```diff
@@ -419,7 +449,7 @@ while (1)
     {
         $config{mode} = 'qps';
         Clear() unless $config{batchmode};
-        print "Queries Per Second [hit q to exit this mode]\n";
+        print "Queries Per Second\n";
```

Comment:
It would make sense to either have `[hit t to exit this mode or q to exit the application]` or to "fix" it so it behave as on the other modes by keeping the ability to use `?` for help or for example to be able to use `M` to call the status mode.